src/apps/simplenavigation/main.cpp

```diff
@@ -394,8 +394,8 @@ std::vector<std::byte> encode() {
     return data;
 }
 
-void decode(const std::vector<std::byte>& data, unsigned int pos) {
-    deserializeObject(data, pos, xform);
+void decode(const std::vector<std::byte>& data, unsigned int apos) {
```

Comment:
`apos`?

Reply:
you have a global pos above, hence I get a shadow warning here...